.github/ISSUE_TEMPLATE/release.md

```diff
@@ -76,11 +70,10 @@ production readiness.
 
 - [ ] Incorporate this tag into `main`'s history by preparing dev on top of it.
 
-  Without checking out main to `main`, perform:
+  Without checking out `main`, perform:
 
   ```shell
-  next=v<NEXT_VERSION>-dev
-  bin/mkrelease --no-tag -b prepare-$next $next
+  bin/mkrelease incorporate
```

Comment:
> Why are we doing that?

Lots of reasons:

* We need to incorporate the LICENSE and config file changes into main for the next release anyway, so every release is going to involve a merge of config file changes, there is no way to make the git history less tangled.
* It makes the git show tags when you browse the log automatically. Without reincorporation you would need to do something like `git log main $(git tag)` if you want to include tags in the log view.
* It makes `git describe` actually do the right thing, otherwise `git describe` will just show that we are progressively further ahead of whatever the last tag that was incorporated into git was.

Reply:
For point 1 -- we could make the config changes on main before branching, couldn't we?

Reply:
We still need to make a commit with some changes. @benesch isn't against merge commits in general, so the fact that this particular merge commit includes a tag seems like the relevant detail, but I don't understand what the actual problem is here.